### PR TITLE
Recover the performance test. 

### PR DESCRIPTION
**Pull request by ShaleXIONG**
Recover the performance test. Run them on CI with a smaller dataset.

### CODE REVIEW

**Comment by veracruz-project-owner** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: Veracruz
* Commit ID: 2d2eb8c08229bec802ab870c6e89c0ee495746ff
* Result: SUCCEEDED
* [Build Logs](https://01evtei465.execute-api.eu-west-2.amazonaws.com/Prod/buildlogs?key=b4f5a87b-eba9-4a35-bdcd-660b42a7c7cd%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: test-collateral/Makefile
Comment (dominic-mulligan-arm) [inline test-collateral/Makefile:107]: Nit: some bizarre indentation here.  Can you fix?

**Comment by dominic-mulligan-arm** [comment]
Modulo that small indentation issue this looks good to me.  Thanks for fixing this, @ShaleXIONG!

File: test-collateral/Makefile
Comment (ShaleXIONG) [reply test-collateral/Makefile]: It was the space vs tab. The tab is interpreted in different lengths. I changed all to space.

**Review by dominic-mulligan-arm — APPROVED** [review]

**Comment by veracruz-project-owner** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: Veracruz
* Commit ID: 7613134fc1a1882c78a388ce95bed316d01fb833
* Result: SUCCEEDED
* [Build Logs](https://01evtei465.execute-api.eu-west-2.amazonaws.com/Prod/buildlogs?key=52ad9988-3145-4d46-ba99-8ce433a77f40%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by veracruz-project-owner** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: Veracruz
* Commit ID: 24e66893de98875b84528d1b0a7311bb472d7967
* Result: SUCCEEDED
* [Build Logs](https://01evtei465.execute-api.eu-west-2.amazonaws.com/Prod/buildlogs?key=590fb19d-70c9-4941-89d7-9e8217d0acc3%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Review by dominic-mulligan-arm — APPROVED** [review]

**Comment by dominic-mulligan-arm** [comment]
LGTM, modulo the rebase onto `main`.  Once that's done, I'll approve again immediately.

**Review by dominic-mulligan-arm — APPROVED** [review]

**Comment by veracruz-project-owner** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: Veracruz
* Commit ID: 3c95494030d5ca4dc27620ba3a1d61655512cf8d
* Result: SUCCEEDED
* [Build Logs](https://01evtei465.execute-api.eu-west-2.amazonaws.com/Prod/buildlogs?key=70196264-027e-4357-a682-5d9059c89042%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by dominic-mulligan-arm** [comment]
+1 reached, merging.

**Pull request merged**
